README.md

```diff
@@ -31,6 +31,21 @@ In your project directory, run `kinvey config` to set-up your project. The CLI w
 
 You can create a one-time session using the `-e` and `-p` (and optionally `--host`) flags. Set these parameters explicitly in your calls if you are unable to init with `kinvey config [host]`.
```

Comment:
This section is actually related to credentials, and is a bit redundant now that you've added a full section on them. I would look at either a) removing it or b) incorporating it into the ### Credentials section as you see fit